examples/pubsub_realtime/README_pubsub_TSN.txt

```diff
@@ -127,7 +127,7 @@ If MAC address is changed, follow INGRESS POLICY steps with dst_mac address to b
 
 To write the published counter data along with timestamp in csv uncomment #define UPDATE_MEASUREMENTS in pubsub_TSN_publisher.c and pubsub_TSN_loopback.c applications
 
-NOTE: The CSV files will be generated in the build directory containing the timestamp and counter values for benchmarking only after terminating both the applications using �Ctrl + C�
+NOTE: The CSV files will be generated in the build directory containing the timestamp and counter values for benchmarking only after terminating both the applications using �Ctrl + C�
```

Comment:
Is this intended?

Reply:
![grafik](https://user-images.githubusercontent.com/3939648/95547280-de41d200-0a02-11eb-97a9-9a6559394514.png)
Might be a bug on the file view.